Survey/Configurations/Plists/Info.plist

```diff
@@ -2,6 +2,12 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>CLIENT_SECRET</key>
+	<string>$(CLIENT_SECRET)</string>
+	<key>CLIENT_ID</key>
+	<string>$(CLIENT_ID)</string>
+	<key>Configuration</key>
+	<string>$(CONFIGURATION)</string>
```

Comment:
I don't see this key being set, is it auto-applied from Configuration's name?

Reply:
Yes, it is 😄